requirements/_main.txt

```diff
@@ -0,0 +1,3 @@
+aiohttp==3.3.2
```

Comment:
could you explain why this and awscli + boto3 were split out into separate files?

Reply:
My motivation is that all that is put through the `extras_require` in the file `setup.py` should be divided. The that is only one way to update or add dependencies.

`There should be one-- and it should be only oneobvious way to do it.`